src/Symfony/Component/ExpressionLanguage/Node/ArrayNode.php

```diff
@@ -58,6 +58,19 @@ public function evaluate($functions, $values)
         return $result;
     }
 
+    public function dump()
+    {
+        $str = '[';
+
+        foreach ($this->getKeyValuePairs() as $pair) {
+            $str .= sprintf('%s => %s, ', $pair['key']->attributes['value'], $pair['value']->dump());
```

Comment:
No. I said that because the parser distinguishes hashes and lists in ExpressionLanguage (as does Twig), but your dumping does not